components/actionbutton/stories/actionbutton.stories.js

```diff
@@ -110,6 +140,82 @@ Quiet.parameters = {
 	},
 };
 
+/**
+ * An action button can have a hold icon (a small corner triangle). This icon indicates that holding down the action button for a short amount of time can reveal a popover menu, which can be used, for example, to switch between related actions.
+ */
+
+export const HoldIcon = IconOnlyOption.bind({});
+HoldIcon.args = ["!dev"];
+HoldIcon.parameters = {
+	chromatic: {disableSnapshot: true},
+};
+
+export const StaticWhiteDocs = TreatmentTemplate.bind({});
+StaticWhiteDocs.tags = ["!dev"];
+StaticWhiteDocs.args = {
+	staticColor: "white",
+};
+
+StaticWhiteDocs.storyName = "Static white";
+
+StaticWhiteDocs.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const StaticWhiteQuiet = TreatmentTemplate.bind({});
+StaticWhiteQuiet.tags = ["!dev"];
+StaticWhiteQuiet.args = {
+	staticColor: "white",
+	isQuiet: true,
+};
+
+StaticWhiteDocs.storyName = "Static white (quiet)";
```

Comment:
```suggestion
StaticWhiteQuiet.storyName = "Static white (quiet)";
```
It looks like a couple of the static color story names got mixed up.